### PR TITLE
Express config

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Skeletal layout for Express ORM and Express server

### CODE REVIEW

**Comment by patchandpray** [comment]
Nice work!

Can you put all these files under a directory named: howto-kassandra-express-orm with a README.md explaining what this is, what it does, what people can use it for and link to it from the main README.md

File: src/database.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    key:["name"]
+});
+
+MyModel.syncDB(function(err, result) {
+    if (err) throw err;
+   // creates migrations and syncs it with the DB
+});
+
+var instance = new models.instance.Person({
+    name: "Abhijith",
```
Comment (shreemaan-abhishek) [inline src/database.js:36]: I suggest using Environment vars for the personal details like name and stuff. What do you think @AbhijithGanesh @patchandpray?

Comment (AbhijithGanesh) [reply src/database.js:36]: It was an example for the field of name :). It is a fixture in this case and I don't see why we should use a env var cuz this isn't a secret. I just put that as an example. I didn't want to use John Doe

**Comment by AbhijithGanesh** [comment]
@patchandpray can you merge this?

File: src/database.js (hunk 1 above)
Comment (patchandpray) [reply src/database.js:36]: When we mature we should definitely go for env vars. For now this as an example is fine :)

**Review by patchandpray — APPROVED** [review]

**Pull request merged**